jobs/JGDAS_ATMOS_GEMPAK_META_NCDC

```diff
@@ -69,15 +69,15 @@ export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
 ##############################################
 # Define COM directories
 ##############################################
-export COMIN=${COMIN:-$(compath.py -o ${NET}/${gfs_ver}/${RUN}.${PDY})/${cyc}/$COMPONENT/gempak}
-export COMINgdas=${COMINgdas:-$(compath.py -o ${NET}/${gfs_ver}/${RUN})}
+export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${gfs_ver})/${RUN}.${PDY}/${cyc}/$COMPONENT}
+export COMINgdas=${COMINgdas:-$(compath.py ${envir}/${NET}/${gfs_ver}/${RUN})}
 export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${gfs_ver}/${RUN}.${PDY})/${cyc}/$COMPONENT/gempak/meta}
 export COMOUTncdc=${COMOUTncdc:-$(compath.py -o ${NET}/${gfs_ver}/${RUN}.${PDY})/${cyc}/$COMPONENT}
 
 # export ukmet_ver=v2.2.0
-export COMINukmet=${COMINukmet:-$(compath.py -o ukmet/${ukmet_ver}/ukmet)}
+export COMINukmet=${COMINukmet:-$(compath.py ${envir}/ukmet/${ukmet_ver})}
```

Comment:
I think it should be:
export COMINukmet=${COMINukmet:-$(compath.py ${envir}/ukmet/${ukmet_ver})/ukmet}
Because later in the scripts:
export HPCUKMET=${COMINukmet}.${PDY}

Same for COMINecmwf in the next line.

Reply:
compath.py call closed at ${ukmet_ver}, then append "/ukmet".

Reply:
Fixed, thanks!